include/seastar/http/request.hh

```diff
@@ -109,12 +109,12 @@ struct request {
     }
 
     /**
-     * Search for the first header of a given name
-     * @param name the header name
-     * @return a pointer to the header value, if it exists or empty string
+     * Search for the last query parameter of a given key
```

Comment:
`query_parameters` is an `unordered_map`, so there's only one match, but good call fixing the comment.

Reply:
Yeah, I guess search is the wrong word here. Nevertheless I wanted to call out that if there are multiple query parameters in the url with the same key (eg. `/a?duplicated_param=x&duplicated_param=y`) then the last one is taken based on the order in which the values are inserted into the map here: https://github.com/redpanda-data/seastar/blob/3d74dce9f5bed1d30742b61b058db7d4c44c7cc8/src/http/request.cc#L85-L99

Reply:
Oh, I see. It depends on the entire path of how we got here, not just the method in isolation.